FewCore/Element.swift

```diff
@@ -116,7 +118,9 @@ public class Element {
 				compareAndSetAlpha(view, alpha)
 			}
 			
-			realizedSelf.element = self
+			if old.frame.size != frame.size {
```

Comment:
Oh you're totally right.

But I *think* we do need to handle the case where the element doesn't have a specific width/height at this time, meaning its layout with be determined by flexbox.

I think something like this would do the trick:

```
if old.frame.size != frame.size && !(frame.size.width.isNaN && frame.size.height.isNaN) {
```